Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListCollectionViewLayout.swift

```diff
@@ -6,11 +6,19 @@ import Foundation
 import UIKit
 
 open class ChatChannelListCollectionViewLayout: UICollectionViewFlowLayout {
-    override public required init() {
+    override public init() {
         super.init()
+        commonInit()
     }
     
     public required init?(coder: NSCoder) {
         super.init(coder: coder)
+        commonInit()
+    }
+    
+    private func commonInit() {
+        estimatedItemSize = UICollectionViewFlowLayout.automaticSize
```

Comment:
This is not entirely true. We know that the width will always be the full width, only the heigh is automatic.

Reply:
Good point! I think we can set `estimatedItemSize` in `prepare` to ensure we get the width from collection view.